src/lib/jsonrpc.ml

```diff
@@ -82,24 +82,20 @@ let new_id =
     !count
 
 let string_of_call ?(version= V1) call =
-  let json =
+  let jsontmp =
```

Comment:
You can use json both here and below, the next binding will shadow it if needed, but it’s not an essential change